chatcommands.py

```diff
@@ -555,6 +555,7 @@ def welcome(msg, other_user):
     :param other_user:
     :return: A string
     """
+    other_user = regex.match(r'[^@\s]{3,}', other_user)
```

Comment:
Run this in a Python terminal, find out what you get. I'm getting `other_user` is always `None`.